config/database.yml

```diff
@@ -21,9 +21,10 @@ test:
 
 deploy_default: &deploy_default
   <<: *default
-  host: <%= ENV["DATABASE_HOST"] %>
-  username: <%= ENV["DATABASE_USER"] %>
-  password: <%= ENV["DATABASE_PASSWORD"] %>
+  host: <%= ENV["DATABASE_HOST"] if ENV["DATABASE_USER"] %>
+  username: <%= ENV["DATABASE_USER"] if ENV["DATABASE_USER"] %>
+  password: <%= ENV["DATABASE_PASSWORD"] if ENV["DATABASE_PASSWORD"] %>
+  url: <%= ENV["DATABASE_URL"] %>
```

Comment:
We need the `DATABASE_URL` for heroku env

Reply:
should these also be defined in the dot env file?